nemo/collections/asr/modules/audio_preprocessing.py

```diff
@@ -199,6 +203,7 @@ def __init__(
         dither=1e-5,
         pad_to=16,
         frame_splicing=1,
+        stft_exact_pad=False,
```

Comment:
Can we remove `stft_exact_pad` here?

Reply:
SqueezeWave needs slightly different padding than the default STFT modules that we have